config/config-api/test/com/thoughtworks/go/config/MergeCruiseConfigTest.java

```diff
@@ -299,56 +298,22 @@ public void addPipelineWithoutValidation_shouldAddPipelineToMain()
         assertThat(cruiseConfig.hasPipelineNamed(new CaseInsensitiveString("pipe3")), is(true));
 
     }
+
     @Test
-    public void addPipelineWithoutValidation_shouldFailToAddPipelineWhenItExistsInPartialConfig()
```

Comment:
These were testing the behaviour of methods such as `addPipeline`, `addPipelineWithoutValidation` etc in `MergeStrategy`, which are deleted as a part of this PR, hence deleting the corresponding tests.